admin/build_targets/centos-7/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN ["yum", "install", "--assumeyes", "git", "ruby-devel", "python-devel", "pyth
 # See https://github.com/jordansissel/fpm/issues/611
 RUN ["gem", "install", "fpm"]
 
-RUN ["pip", "install", "twisted==14.0.2", "characteristic==14.1.0", "virtualenv==12.0.5"]
+RUN ["pip", "install", "twisted==14.0.2", "characteristic==14.1.0", "virtualenv==12.0.5", "effect=0.1a13"]
```

Comment:
Also, I wonder if we should update the Twisted version requirement to `15.0.0`  while we're here?

I wonder how we could refactor these requirements so that they can be shared between all the platform Dockerfiles?